### DISCUSSION TITLE
Support for deploy Karavan in a kubernetes cluster

### DISCUSSION THREAD

**Discussion by noir-madlax** [apache/camel-karavan#741] [Q&A] [answered]
I have already deployed Karavan in Minikube, and everything is running normally. 
Now I want to deploy it in my own kubernetes cluster. Are there any minimum version requirements for kubernetes? 
Also, what other points should I pay attention to?
I would be very grateful if anyone could tell me, thank you.

**Comment by mrinalsharma** [comment, marked as answer]
please review https://github.com/apache/camel-karavan/tree/main/karavan-cloud/AWS

**Reply by noir-madlax** [reply]
Thank you very much! I have successfully deployed it on my self-hosted Kubernetes cluster.
